tests/utils/Constants.sol

```diff
@@ -15,6 +15,7 @@ abstract contract Constants {
     uint256 public constant MIN_FEE_WEI = (1e18 * MIN_FEE_USD) / 3000e8; // at $3000 per ETH price
     uint128 public constant START_AMOUNT = 100e18;
     UD2x18 public immutable START_PERCENTAGE = (ud(START_AMOUNT).div(ud(CLAIM_AMOUNT)).intoUD2x18());
+    uint128 internal constant VCA_FULL_AMOUNT = CLAIM_AMOUNT;
```

Comment:
Has to be removed

Reply:
I added it because I am using it in `calculateClaimAmount.t.sol` and `calculateForgoneAmount.t.sol`. Please have a look and let me know if it makes sense. I could have used the `CLAIM_AMOUNT` there but `VCA_FULL_AMOUNT` makes more sense since we are already using it in `claim`.

Reply:
Oh, right